packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -390,6 +391,11 @@ export function TransferPanelMain({
 
   const [loadingMaxAmount, setLoadingMaxAmount] = useState(false)
   const [withdrawOnlyDialogProps, openWithdrawOnlyDialog] = useDialog()
+  const [oneNovaTransferDialogProps, openOneNovaTransferDialog] = useDialog()
+  const [
+    oneNovaTransferDestinationNetworkId,
```

Comment:
We can't use `to` state as a prop for `destinationChainId` in `OneNovaTransferDialog`. This is because before `to` changes to a new network, some metamask actions can be triggered such as switching network in the wallet. That makes UX quite bad.